.env-development

```diff
@@ -0,0 +1,3 @@
+GOOGLE_CLIENTID=214670141134-o0ffgiatndiplsm6nkargad8qnacccoa.apps.googleusercontent.com
```

Comment:
that's kind of secretive, please don't submit this file

Reply:
I removed it, but this file is also in the master branch, and in the 2018 repo as well